junifer/datagrabber/aomic.py

```diff
@@ -0,0 +1,354 @@
+"""Provide concrete implementations for AOMIC1000 data access."""
```

Comment:
The idea was to create an `aomic` module (folder) and place the 3 AOMIC datagrabber inside, each one in a different file.

Reply:
ah ok makes sense